android/README.md

```diff
@@ -138,4 +138,4 @@ The result is in a row corresponding to the column named `result`. The value wil
 
 Note that a pack can be added to either the main WhatsApp app, WhatsApp Business, or both. It is recommended you continue to present a button to add the pack to WhatsApp if the sticker pack is not added to one or more of the apps. Refer to the class [WhitelistCheck](app/src/main/java/com/example/samplestickerapp/WhitelistCheck.java) for sample logic.
 
-Your app can only query whether the packs it provides have been added and it can't check for information about sticker packs from other apps.
\ No newline at end of file
+Your app can only query whether the packs it provides have been added and it can't check for information about sticker packs from other apps.
```

Comment:
what changed here?

Reply:
I actually don´t know but I think the line ending changed because I edited it in GitHub Web. So there´s no visible change.